leetcodePython/PrefixSum/prefixsum_2021.py

```diff
@@ -0,0 +1,22 @@
+import collections
+from typing import List
+
+
+class Solution:
+    def brightestPosition(self, lights: List[List[int]]) -> int:
+        bright_change = collections.defaultdict(int)
+        for pos, range in lights:
+            start_pos = pos - range
+            end_pos = pos + range + 1
+            bright_change[start_pos] += 1
+            bright_change[end_pos] -= 1
```

Comment:
The only difference is when a light start and when a light end. We record these positions